pkg/apis/batch/v1alpha1/job.go

```diff
@@ -334,6 +340,11 @@ type JobStatus struct {
 	// +patchMergeKey=status
 	// +patchStrategy=merge
 	Conditions []JobCondition `json:"conditions,omitempty" patchStrategy:"merge" patchMergeKey:"status" protobuf:"bytes,13,rep,name=conditions"`
+
+	// Represents time when the job controller started processing a job.
+	// It is represented in RFC3339 form and is in UTC.
+	// +optional
+	StartTime *metav1.Time `json:"startTime,omitempty" protobuf:"bytes,14,opt,name=startTime"`
```

Comment:
There is a field in the status of vcjob that indicates the running time. Can we use it?

Reply:
I think we can use `creationTimestamp`,
kubernetes job can suspend, so creationTimeStep can not used by them. Because when a job is created, it may be a suspend state. Volcano job does not have suspend.

But there's still a choice to be made.
Should we calculate the creation time or the run time?

Reply:
Ok，in case of we support suspend semantic in the future, it's a good choice to use startTime.